src/ui/CustomstyleDropDownCommand.tsx

```diff
@@ -49,7 +49,9 @@ export class CustomstyleDropDownCommand extends React.PureComponent<{
     if (this.hasRuntime) {
       getStylesAsync().then((result) => {
         if (result) {
+          if (firstLoad === 0) {
```

Comment:
Why do we need this check? result should always be the latest version of styles, so I'd expect this to only be introducing an update failure bug.